documentation/library_guide/kernel_templates/esimd/radix_sort_by_key.rst

```diff
@@ -0,0 +1,361 @@
+| ``q``                                         | The SYCL* queue where kernels are submitted.                        |
++-----------------------------------------------+---------------------------------------------------------------------+
+|                                               |                                                                     |
+|                                               | The sequences to apply the algorithm to.                            |
+| - ``keys_first``, ``keys_last``,              | Supported sequence types:                                           |
+|   ``values_first`` (1),                       |                                                                     |
+| - ``keys``, ``values`` (2),                   | - :ref:`USM pointers <use-usm>` (1,3),                              |
+| - ``keys_first``, ``keys_last``,              | - :ref:`oneapi::dpl::begin and oneapi::dpl::end                     |
+|   ``values_first``, ``keys_out_first``,       |   <use-buffer-wrappers>` (1,3).                                     |
+|   ``values_out_first`` (3)                    | - ``sycl::buffer`` (2,4),                                           |
+| - ``keys``, ``values``,                       | - :ref:`oneapi::dpl::experimental::ranges::views::all               |
+|   ``keys_out``, ``values_out`` (4).           |   <viewable-ranges>` (2,4),                                         |
+|                                               | - :ref:`oneapi::dpl::experimental::ranges::views::subrange          |
+|                                               |   <viewable-ranges>` (2,4).                                         |
+|                                               |                                                                     |
+|                                               |                                                                     |
+|                                               |                                                                     |
++-----------------------------------------------+---------------------------------------------------------------------+
+| ``param``                                     | A :doc:`kernel_param <../kernel_configuration>` object.             |
+|                                               | Its ``data_per_workitem`` must be a positive multiple of 32.        |
+|                                               |                                                                     |
+|                                               |                                                                     |
++-----------------------------------------------+---------------------------------------------------------------------+
+
+
+**Type Requirements**:
+
+- The element type of sequence(s) to sort must be a C++ integral or floating-point type
+  other than ``bool`` with a width of up to 64 bits.
+
+.. note::
```

Comment:
This formatting is odd and not rendering in Note form. Should this be a note or a header?

Reply:
I think that the Current Limitations sub-header was confusing me. I built the docs and can see that it does render correctly, so this is a non-issue that you can ignore.